.eslintrc.js

```diff
@@ -10,6 +10,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+'use strict';
+
```

Comment:
Why does it complain here? It's its own config file and it never needs `use strict` in practice? And I have never seen it complaining here, even though it did complain properly if `use strict` was missing in "real" js code.

Reply:
The reason is eslint v7, it seems.

Reply:
Ok. Well, it's a copy paste file, so no reason to waste time on it and just accept :)